python/mxnet/gluon/block.py

```diff
@@ -1355,9 +1359,18 @@ def export(self, path, epoch=0, remove_amp_cast=True):
                     else:
                         arg_dict['aux:%s'%name] = param._reduce()
         save_fn = _mx_npx.save if is_np_array() else ndarray.save
-        params_filename = '%s-%04d.params'%(path, epoch)
-        save_fn(params_filename, arg_dict)
-        return (sym_filename, params_filename)
+        params_filename = '%s-%04d.params'%((path if path is not None else ""), epoch)
+
+        if path is not None:
+            save_fn(params_filename, arg_dict)
+            return (sym_filename, params_filename)
+
+        if remove_amp_cast:
+            handle = SymbolHandle()
+            import ctypes
+            check_call(_LIB.MXSymbolRemoveAmpCast(sym.handle, ctypes.byref(handle)))
+            sym = type(sym)(handle)
+        return sym, arg_dict
```

Comment:
What about aux?

Reply:
`arg_dict` here contains aux as well

Reply:
oh ok, saw that in the other comment. makes sense. although the variable names are confusing: `arg_names`, `aux_names` and then putting both args/aux into `arg_dict` :-) but thats not your code here...